charter/index.html

```diff
@@ -167,40 +172,41 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
+        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities and others (e.g. the <a href="https://www.w3.org/community/fed-id/">Federated Identity CG</a> and the <a href="https://www.w3.org/community/socialcg/">Social Web Incubator CG</a>).</span></p>
 
-        <p>The <cite><a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a></cite> has been incubating <a href="https://solidproject.org/TR/">Technical Reports</a> since 2018 towards that end: <q cite="https://www.w3.org/groups/cg/solid">to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces.</q></p>
+        <p>The wider Solid community has developed a <a href="https://solidproject.org/developers/tools/">suite of implementations, tools, and libraries</a> for developers, as well as <a href="https://solidproject.org/apps">applications</a> for users.:
+        <span class=new>See a separate <a href="https://timbl.solidcommunity.net/2023/03%20EWADA/TAGExplainerforSolid.html">explainer</a> for more detailed description of the Solid approach.</span>
+        </p>
 
-        <p>The wider Solid community has developed a <a href="https://solidproject.org/developers/tools/">suite of implementations, tools, and libraries</a> for developers, as well as <a href="https://solidproject.org/apps">applications</a> for users.</p>
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
+
         <p>
-                The design approach for the <a href="https://solidproject.org/TR/protocol">Solid Protocol</a> will continue substantial efforts undertaken previously in the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> to specify a set of interoperable protocol standards for enabling the use and development of Solid servers and applications across a <a href="https://github.com/solid/user-stories">wide range of use cases</a>.
+          <span class=new>The scope of the Working Group is to define a web protocol between client-side application, on the one hand, and storage and identity server, on the other hand, allowing</span> users to keep authority over their data, identity, and privacy.
+          This protocol must ensure interoperability between compliant applications and various storage and identity server,
+          thus giving users the freedom to select the applications and services that suit their needs.
         </p>
 
+
         <p>
-                The Working Group will:
+        Based on the <a href="https://github.com/solid/user-stories">user cases</a> identified by the Solid project, potentially extended by Working Group participants, the Working Group will:
```

Comment:
The use cases are in a bad state, not maintained and woefully out of date.  Granted folks might not click through, but consider either making use cases more attractive, or removing this.

Reply:
The use cases are there to serve as input for discussions. They are not in a "bad state" but surely they can be always better organised or be more "attractive". It serves its purpose for the charter in that the CG has done some homework. Removing goes counter the main takeaway here.

Reply:
Do you think the use cases are in a good state?

https://github.com/solid/user-stories

If anyone clicks through, it gives a fairly unintuitive, wall of text, imho.

If anyone reading the charter works out that the user stories are in the issues tab, they will see this:

https://github.com/solid/user-stories/issues

At present time, it looks like a largely abandoned piece of work, with barely much activity over many years, and some people asking what's implemented without anyone having an answer.

It's OK if no one is going to look at it, but if someone does, would you consider it to be attractive, and well thought through?